scripts/linter/missing-sections.js

```diff
@@ -0,0 +1,52 @@
+const visit = require("unist-util-visit");
+
+const { hasRecipe } = require("./recipes");
+const { isHeadingLevel2, remarkToSlug } = require("./heading-utils");
+
+const ruleId = "stumptown-linter:missing-section";
+
+/**
+ * Re
+ */
+function requiredSections(recipe) {
+    const proseRequiredIngredients = recipe.body.filter(
+        ingredient =>
+            ingredient.startsWith && // ignore non-string ingredients
```

Comment:
Why not `typeof(ingredient) === 'string'`?

Reply:
Yes, I've been there. I guess my feeling is that if `typeof(ingredient) === 'string'` doesn't work properly, that's JavaScript's problem not mine :).